packages/patternfly-4/content/design-guidelines/usage-and-behavior/charts/design.md

```diff
@@ -62,20 +64,58 @@ Stacked bar charts describe totals while allowing a degree of internal breakdown
 <!--#### React component
 [Stacked bar chart](/documentation/react/components/stackchart)-->
 
-### Donut chart
+### Donut charts
 Donut charts represent relative amounts that must add up to 100%.
 
 #### Usage
-The most common use cases for donut charts are to show progress completion as a percentage, show utilization for an object as a percentage and show the relationship of a set of values to a whole. Use donut charts to compare an individual section to the whole. As said previously, the entire donut should add up to 100% and the fill represents a percentage of that. There should be no more than six categories in total, similar to pie charts We recommend that you use the [Colors for charts](/design-guidelines/styles/chart-colors) to represent your data when thresholds are not present. Consider using a [Bar chart](#bar-chart) if you would like to compare one category to another.
+These are the most common use cases for donut charts:
+  - Showing the relationship of a set of values to a whole
+  - Showing progress completion as a percentage
+  - Showing utilization for an object as a percentage (with or without thresholds)
 
-#### Example
-![Donut chart](./img/donutchart.png)
+Donut charts can use percentages or whole numbers to compare parts to the whole. Consider using a [Bar chart](#bar-chart) if you would like to compare one category to another. Decide whether percentages or whole numbers is most compatible with your use case and make sure to be consistent throughout. We recommend displaying these values within the donut, see [Donut utilization](#donut-utilization) for styling. Don’t use more than six categories in total. We recommend that you use the [Colors for charts](/design-guidelines/styles/chart-colors) guidelines to represent your data when thresholds are not present.
+
+![Donut chart](./img/donut chart.png)
+
+1. **Segment fill**: We recommend using [Colors for charts] (/design-guidelines/styles/chart-colors) for different items within the donut chart.
+2. **Segment padding**:  Always provide 3px of padding between segments.
+3. **Chart tooltip**:  A tooltip will appear upon hover that states the segment name and value. Depending on the type of donut, the information may change.
+  - **Donut**: We recommend stating the segment name and the value being represented. For example, if the segment represents “Bugs,” and the value being represented is 25, your chart tooltip would state, “Bugs - 25.”
```

Comment:
Perhaps recommend "Bugs: 25" vs "Bugs - 25". It's possible to have negative values and at a glance I read this as -25.

Reply:
If we make the Name: Value change here we'll have to make it everywhere. Which I'm okay with but will be updating other images as well

Reply:
What do you think @stacymcauliffe ? If it's just me who's reading it this way, then it might not be worth all the work. I did not feel this was a problem when I looked at the images, only reading the text, so maybe it's OK.

Reply:
@mcarrano @stacymcauliffe Agreed and changed to use ":"